cli/validate.go

```diff
@@ -58,16 +70,16 @@ func validateRun(cmd *cobra.Command, args []string) {
 	for _, request := range requests {
 		ok, err := controller.ValidateRequest(request)
 		if ok {
-			newLogger.Info(newCtx, "Group '%v' is valid.", request.Group)
+			fmt.Printf("Group '%v' is valid.\n", request.Group)
```

Comment:
yes its intended. @joe and I talked and feel that printing here is the nicer output.